infrastructure/templates/public-api/components/functionApp.bicep

```diff
@@ -143,14 +169,47 @@ resource functionApp 'Microsoft.Web/sites@2023-01-01' = {
       preWarmedInstanceCount: preWarmedInstanceCount ?? null
       netFrameworkVersion: '8.0'
       linuxFxVersion: appServicePlanOS == 'Linux' ? 'DOTNET-ISOLATED|8.0' : null
+      keyVaultReferenceIdentity: userAssignedManagedIdentityParams != null ? userAssignedManagedIdentityParams!.id : null
     }
+    keyVaultReferenceIdentity: userAssignedManagedIdentityParams != null ? userAssignedManagedIdentityParams!.id : null
+    publicNetworkAccess: publicNetworkAccessEnabled ? 'Enabled' : 'Disabled'
   }
   tags: tagValues
 }
 
+resource azureStorageAccounts 'Microsoft.Web/sites/config@2021-01-15' = {
+   name: 'azurestorageaccounts'
+   parent: functionApp
+   properties: reduce(azureFileShares, {}, (cur, next) => union(cur, {
+     '${next.storageName}': {
+       type: 'AzureFiles'
+       shareName: next.fileShareName
+       mountPath: next.mountPath
+       accountName: next.storageAccountName
+       accessKey: next.storageAccountKey
+     }
+   }))
+}
+
+resource slot1StorageAccountFileShare 'Microsoft.Storage/storageAccounts/fileServices/shares@2023-01-01' = {
+  name: '${durableManagementStorageAccountName}/default/${fullFunctionAppName}1'
+  dependsOn: [
+    durableManagementStorageAccountModule
+  ]
+}
+
+resource slot2StorageAccountFileShare 'Microsoft.Storage/storageAccounts/fileServices/shares@2023-01-01' = {
+  name: '${durableManagementStorageAccountName}/default/${fullFunctionAppName}2'
```

Comment:
Same as above, I don't understand why the name here is `durableManagementStorageAccountName` rather than `slot2StorageAccountName`?

Reply:
As above